### PR TITLE
Feat(web): Add Utilities classes for display, margins, paddings and t…

### PR DESCRIPTION
**Pull request by crishpeen**
…ext align #DS-166

### CODE REVIEW

**Comment by literat** [comment]
I am reluctant to use LGTM. But I think I will used to it. I am not certain if I can decipher all of the utilities at first sight.

File: packages/web/src/settings/_utilities.scss [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        responsive: true,
+        property: padding-block,
+        class: py,
+        values: tokens.$spaces,
+    ),
+    'text-align': (
+        responsive: true,
+        property: text-align,
+        class: text,
+        values: left right center justify,
```
Comment (adamkudrna) [inline packages/web/src/settings/_utilities.scss:118]: I'd vote against `justify`. It's not a good choice on the web and that's why it's [not part of Bootstrap](https://getbootstrap.com/docs/5.0/utilities/text/#text-alignment) either.

File: packages/web/src/tools/_utilities.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
+@use 'sass:list';
```
Comment (adamkudrna) [inline packages/web/src/tools/_utilities.scss:1]: Please add a credit to Bootstrap here and in the calling file, we want to be the nice guys 🙂.

**Review by adamkudrna — CHANGES_REQUESTED** [review]

File: examples/web/index.html [hunk 3]
```diff
@@ -11,37 +11,17 @@
     <link rel="stylesheet" href="built/foundation.min.css">
     <link rel="stylesheet" href="built/components.min.css">
     <link rel="stylesheet" href="built/helpers.min.css">
+    <link rel="stylesheet" href="built/utilities.min.css">
 
     <style>
       body {
-        margin: 0;
-        padding: 0;
         font-family: Inter, sans-serif;
```
[the remark this answers is not in the document]
Comment (crishpeen) [reply examples/web/index.html:18]: Yea, sure, why not :)

File: packages/web/src/tools/_utilities.scss (hunk 2 above)
Comment (crishpeen) [reply packages/web/src/tools/_utilities.scss:1]: It is rather inspired by RUI, but I will mention them both.

**Pull request merged**